test/OpenSTUtility.js

```diff
@@ -363,7 +360,7 @@ contract('OpenSTUtility', function(accounts) {
 			var redeemReturns = await openSTUtility.redeem.call(checkBtUuid, redeemAmountUT, 2, redeemBeneficiary, lockR.l, { from: redeemer });
 
             // call block number is one less than send block number
```

Comment:
Global comment for where this inline comment appears: I think this inline comment no longer applies